src/app/new-auth/login/login.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+
+@Component({
+  selector: 'clark-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit{
+
+  loginFailure = false;
+  constructor(public authValidation: AuthValidationService) { }
+
+  ngOnInit(): void {
+  }
+/**
+ * TO-DO: implement this method
+ *
+ * @param f form data
+ */
+  public submit(f: NgForm): void {
```

Comment:
```suggestion
  public submit(form: NgForm): void {
```
Just a suggestion on better naming.